### DISCUSSION TITLE
Wording at the credits section

### DISCUSSION THREAD

**Discussion by humrochagf** [tenacityteam/tenacity-legacy#479] [Q&A] [answered]
Hi, I started to do some translation of the app to Brazilian Portuguese.
But, different than English, Portuguese has grammatical gender.

So, a pattern that I came up with to achieve gender neutrality in the contribution section was to, instead of describing the person describe their contribution:

Tenacity contributor -> Tenacity contribution
Audacity developer -> Audacity development
Audacity co-founder and developer -> Audacity co-founding and development

This approach works at least for the languages with grammatical gender that I'm familiar with (Portuguese, French and Spanish )

Maybe, adopting this wording in the English version would help achieve gender neutrality at other translations as well, what do you think about it?

**Comment by nbsp** [comment, marked as answer]
I recall bringing this issue up on the IRC channel myself (Hebrew also has grammatical gender), but got no response. 
I think that this is a great solution for languages with grammatical gender, but maybe not worth changing in the English one.

**Reply by poyribn** [reply]
Yes, in English this change would be confusing.

**Comment by petersampsonaudacity** [comment]
> So, a pattern that I came up with to achieve gender neutrality in the contribution section was to, instead of describing the person describe their contribution:
> 
> Tenacity contributor -> Tenacity contribution
> Audacity developer -> Audacity development
> Audacity co-founder and developer -> Audacity co-founding and development

In English (UK and American) contributor, developer founder are all gender-neutral.

**Reply by humrochagf** [reply]
Yeah, I thought it more as a guideline for other languages, but this post in itself could already work as a first discussion to provide some tips for translators. 🙂
